test/Interop/Cxx/stdlib/avoid-redundant-copies.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift -enable-experimental-feature AddressableParameters)
```

Comment:
I don't have a strong preference here, but I'm curious to hear your thoughts around pros and cons of a run time test vs checking SIL or IR output